ERCS/erc-7291.md

```diff
@@ -4,7 +4,7 @@ title: Purpose bound money
 description: An interface extending ERC-1155 to implement purpose-bound money, a hybrid form of programmable payment and money.
```

Comment:
```suggestion
description: An extension to ERC-1155 implementing a hybrid form of programmable payment and money.
```

This has a bit less fluff, but you can probably expand a little more, since you're most familiar with this proposal. The goal of the description is to add detail over what is introduced in the title.